src/test/java/courseManager/CourseManagerTest.java

```diff
@@ -1,8 +1,10 @@
 package courseManager;
 
+import entities.assessment.Assessment;
+import entities.course.Course;
```

Comment:
This file is behind main. sami's pull request was approved recently, try merging main and hopefully the issue will be resolved